public/src/pages/marketplace/Partners/Partners.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { FC, useEffect, useState } from 'react';
+import { DOMParser } from 'xmldom';
+
+import OwlCarousel from 'react-owl-carousel';
+import 'owl.carousel/dist/assets/owl.carousel.css';
+import 'owl.carousel/dist/assets/owl.theme.default.css';
+
+import { searchPartners, partnerLogin } from '../../../api/httpClient';
+import { Partner } from '../../../interfaces/Partner';
+
+export const Partners: FC = () => {
+  const PARTNER_DEFAULT_USERNAME = 'walter100';
+  const PARTNER_DEFAULT_PASSWORD = 'Heisenberg123';
+
+  const [partners, setPartners] = useState<Array<Partner>>([]);
+
+  const partnerNameSearchEP = () => {
+    // XPATH injection string detection
+    searchPartners('').then((data) => {
+      const partnersList: Array<Partner> = [];
+
+      const xmlDoc = new DOMParser().parseFromString(data, 'text/xml');
+
+      if (!xmlDoc) {
+        partnersList.push({ name: 'Failed loading name', photoUrl: '' });
+        setPartners(partnersList);
+        return;
+      }
+
+      const partnerNameTags = xmlDoc.getElementsByTagName('name');
+
+      for (const nameTag of Array.from(partnerNameTags)) {
+        const name = nameTag.textContent || 'Error in loading name';
+        const photoUrl =
+          'assets/img/partners/' +
+          name.toLowerCase().replace(' ', '-') +
+          '.jpg';
+        partnersList.push({ name: name, photoUrl: photoUrl });
+      }
+
+      setPartners(partnersList);
+    });
+  };
+
+  const partnerLoginEP = () => {
```

Comment:
Here too, `loginPartner` or similar would be better, as this function does not implement an endpoint